webui/react/src/components/ExperimentInfoBox.tsx

```diff
@@ -0,0 +1,76 @@
+import { Button } from 'antd';
+import React from 'react';
+
+import Badge, { BadgeType } from 'components/Badge';
+import Link from 'components/Link';
+import { formatDatetime } from 'utils/date';
+import { humanReadableFloat } from 'utils/string';
+
+import { Checkpoint, CheckpointState, ExperimentDetails } from '../types';
+
+import css from './ExperimentInfoBox.module.scss';
+
+interface Props {
+  experiment: ExperimentDetails;
+}
+
+const pairRow = (label: string, value: React.ReactNode | undefined): React.ReactNode => {
+  return (
+    <tr key={label}>
+      <td className={css.label}>{label}</td>
+      <td>
+        {[ 'string', 'number' ].includes(typeof value) ?
+          <span>{value}</span> :
+          <>
+            { value }
+          </>
+        }
+      </td>
+    </tr>
+  );
+};
+
+const InfoBox: React.FC<Props> = ({ experiment: exp }: Props) => {
+  // CHECK orderfactor
+  const orderFator = exp.config.searcher.smallerIsBetter ? 1 : -1;
+  const sortedValidations = exp.validationHistory
+    .filter(a => a.validationError !== undefined)
+    .sort((a, b) => (a.validationError as number - (b.validationError as number)) * orderFator);
+
+  const bestVal = sortedValidations[0]?.validationError;
```

Comment:
**question:** I think this will fail if `sortedValidations` is empty, null or undefined.

Reply:
hmm if I set up the types correctly sortedValidations has to at least be an empty array :thinking:

Reply:
yeah you're right 👍